### ISSUE TITLE
Too many open files

### ISSUE THREAD

**Issue by conradludgate** [mozilla/cargo-vet#204]
I work on a project with ~600 dependencies!!!

I've just pulled the latest cargo-vet and run `cargo vet suggest`

```
> cargo vet suggest
ERROR error diffing data-encoding:2.3.2 Too many open files (os error 24)
ERROR error diffing derive_more:0.99.17 Too many open files (os error 24)
ERROR error diffing deadpool:0.9.5 Too many open files (os error 24)
ERROR error diffing event-listener:2.5.2 Too many open files (os error 24)
ERROR error diffing generic-array:0.14.5 Too many open files (os error 24)
ERROR error diffing getrandom:0.2.6 Too many open files (os error 24)
ERROR error diffing glob:0.3.0 Too many open files (os error 24)
ERROR error diffing enum-as-inner:0.4.0 Too many open files (os error 24)
ERROR error diffing digest:0.10.3 Too many open files (os error 24)
...
```

Seems there's no limit on concurrent diffing 😅

**Comment by mystor** [comment]
Oops! Fortunately it's an easy fix, I'll post something in a second.

**Comment by mystor** [comment]
@conradludgate I hopefully fixed the issue with #205 - please re-open this issue if you're still running into errors.

**Comment by conradludgate** [comment]
Thanks, It seems to be fixed 😊
